XSConsoleUtils.py

```diff
@@ -49,35 +49,36 @@ def __init__(self, *inParams):
         self.stdout = []
         self.stderr = []
         self.called = False
-        
+
     def _NewPipe(self, *inParams):
         if len(inParams) == 1 and isinstance(inParams, (list, tuple)):
                 params = inParams[0]
         else:
             params = inParams
-            
+
         self.pipe = subprocess.Popen(params,
             stdin=subprocess.PIPE,
             stdout=subprocess.PIPE,
             stderr=subprocess.PIPE,
+            text=True,
             close_fds=True)
         self.called = False
-    
```

Comment:
Please review any commits to not intermix removing trailing whitespace with your code changes. While it's possible to ignore such simple changes with tools, everyone reviewing the changes still has to know how to do it to spot your changes in the middle of all that whitespace.

On GitHub, you have to know that you can find the feature to ignore whitespace in the settings gear menu and for `git log -p` you have to know that `git log -pw` ignores whitespace.

Therefore, if you need to change whitespace, then please in a separate PR (or at least a separate commit).

Reply:
> Please review any commits to not intermix removing trailing whitespace with your code changes. While it's possible to ignore such simple changes with tools, everyone reviewing the changes still have to know how to do ignore them in order to spot your change in the middle of all that whitespace.
> 
> On GitHub, you have to know that you can find the feature to ignore whitespace in the settings gear menu and for `git log -p` you have to know that `git log -pw` ignores whitespace.
> 
> Therefore, if you need to change whitespace, then please in a separate PR (or at least a separate commit).

Thank for your suggestion and will keep it in mind.